content/applications/inventory_and_mrp/inventory/product_management/configure/type.rst

```diff
@@ -4,228 +4,349 @@ Product type
+Transfer goods
+~~~~~~~~~~~~~~
 
-   A consumable product was placed in the package, but the **Content** section does not list it.
+*Transfers* are warehouse operations that involve the movement of goods. Examples of transfers
+include
+:doc:`deliveries and receipts <../../shipping_receiving/daily_operations/receipts_delivery_one_step>`,
+as well as :doc:`internal transfers <../../warehouses_storage/replenishment/resupply_warehouses>`
+between warehouses.
 
-If the *Move Entire Package* feature is enabled, moving a package updates the location of the
-contained storable products. However, the location of consumable products are not updated.
+When creating a transfer for tracked products in the **Inventory** app, transfers modify the on-hand
+quantity at each location. For example, transferring five units from the internal location
+`WH/Stock` to `WH/Packing Zone` decreases the recorded quantity at `WH/Stock` and increases it at
+`WH/Packing Zone`.
 
-.. _inventory/product_management/report:
+For untracked products, transfers can be created, but exact quantities at each storage location are
+not tracked.
 
-Inventory report
-----------------
+.. _inventory/product_management/package:
 
-**Only** storable products appear on the following reports.
+Packages
+~~~~~~~~
+
+Both tracked and untracked (non-inventory), products can be placed in :doc:`packages <package>`.
 
-The *stock report* is a comprehensive list of all on-hand, unreserved, incoming, and outgoing
-storable products. The report is only available to users with :doc:`administrator access
-<../../../../general/users/access_rights>`, and is found by navigating to :menuselection:`Inventory
-app --> Reporting --> Stock`.
+However, for non-inventory products, the quantity is not tracked, and the product is not listed in
+the package's :guilabel:`Contents` (which can be accessed by going to :menuselection:`Inventory app
+--> Products --> Packages`, and selecting the desired package).
 
-.. image:: type/stock-report.png
+.. figure:: type/package-content.png
    :align: center
-   :alt: Show stock reporting list found in Inventory > Reporting > Stock.
+   :alt: Show Packages page, containing the package contents list.
 
-The *location report* is a breakdown of each location (internal, external, or virtual) and the
-on-hand and reserved quantity of each storable product. The report is only available with the
-*Storage Location* feature activated (:menuselection:`Inventory app --> Configuration -->
-Settings`), and to users with :doc:`administrator access <../../../../general/users/access_rights>`.
+   An untracked product was placed in the package, but the **Content** section does not list it.
 
-Navigate to the location report by going to :menuselection:`Inventory app --> Reporting -->
-Locations`.
+Additionally, if the *Move Entire Packages* feature is enabled, moving a package updates the
+location of the contained tracked products but not the contained untracked products. To enable this
```

Comment:
```suggestion
location of the contained tracked products, but not the contained untracked products. To enable this
```

Reply:
Leaving out since it's not an independent clause